integration_tests/dbt_project.yml

```diff
@@ -13,12 +13,24 @@ vars:
   apple_search_ads_ad_group_history_identifier: "ad_group_history_data"
   apple_search_ads_ad_group_report_identifier: "ad_group_report_data"
   apple_search_ads_ad_history_identifier: "ad_history_data"
-  apple_search_ads_ad_report_identifier: "ad_report_data"
+  apple_search_ads_ad_level_report_identifier: "ad_report_data"
   apple_search_ads_keyword_history_identifier: "keyword_history_data"
   apple_search_ads_keyword_report_identifier: "keyword_report_data"
   apple_search_ads_search_term_report_identifier: "search_term_report_data"
   apple_search_ads__using_search_terms: True
 
+  apple_search_ads__ad_group_passthrough_metrics:
+    - name: conversions
+    - name: lat_off_installs
+  apple_search_ads__ad_passthrough_metrics:
+    - name: conversions
+      alias: conversions_alias_should_be_included
+
+  # consistency_test_exclude_metrics: ['conversions']
```

Comment:
Will this be necessary post this release? If not, let's remove it.

Reply:
removed